_docs/_latest/configure/scanner/rules.md

```diff
@@ -1068,7 +1068,8 @@ rules:
 * `applies_to`
   * `type`
     * **Description**: The type of resource to apply the rule to.
-    * **Valid values**: Currently only supports `bucket`.
+    * **Valid values**: One of `bucket`, `cloud_sql_instance`, `dataset`, 
```

Comment:
I think it's cloudsqlinstance, kubernetes_cluster instead of cloud_sql_instance and ke_cluster, can you verify them?

Reply:
Verified and updated.